Documentation/states.md

```diff
@@ -1,6 +1,6 @@
 ## Unit states
 
-There are currently three cluster-level states for a unit:
+`fleetctl` uses declarative model which means it sets desired state to units. For example when you wish to start a unit `fleetctl` sets desired state inside etcd storage and `fleet` daemons connected to this storage just perform desired action.  There are currently three cluster-level states for a unit:
```

Comment:
I don't really like leading with fleetctl here; this is a property of the fleet architecture/model, rather than anything fleetctl specific. For example if someone interacts using just the API, this is all still the case

Also, `fleet` should be `fleetd`